bitrise.yml

```diff
@@ -89,7 +105,6 @@ workflows:
           inputs:
             - search_pattern: '*/maestroReport.xml'
             - test_name: Maestro tests
-      - deploy-to-bitrise-io@2: { }
```

Comment:
after-run block already runs this.

Reply:
Not quite! The after-run block looks into a different directory!

Reply:
the previous step, [custom-test-results-export](https://github.com/bitrise-steplib/step-custom-test-results-export/blob/master/step.yml#L4), moves the test results to a directory observed by `deploy-to-bitrise-io`.

the after run block runs `deploy-to-bitrise` already, so this step is duplicated. That's why we're seeing the maestro test results twice:

<img width="1678" alt="image" src="https://user-images.githubusercontent.com/99293320/231590032-79afb0b9-e775-465f-b595-df5f5a3976d1.png">

Reply:
Hmm, that's not how I would have expected things to work, but I'll trust ya!

Reply:
same 🤔